packages/adapter-commons/test/fixture.ts

```diff
@@ -51,7 +50,7 @@ export class MethodBase
   }
 
   async $update(id: NullableId, _data: Data, _params?: AdapterParams) {
```

Comment:
As far as I know, `NullableId` is wrong for `update`. Only `patch` and `remove` are allowed to have a `NullableId`. Am I wrong @daffl?

Reply:
Correct. It should just be `Id`.  If the interface were setup for `NullableId`, it would mean you could completely replace multiple documents with the same `id`, which isn't possible.

Reply:
This was my bad. A bunch of those issues will be fixed in https://github.com/feathersjs/feathers/pull/2906

Reply:
Nice, thanks!